src/Reducer/filterHandler.js

```diff
@@ -0,0 +1,14 @@
+export const FilterHandler = (state, action) => {
+    const initial = {
+        all : ""
+    }
+    switch (action.type) {
+        case 'FILTER_BY_CATEGORY':
+            return{...state, category:action.payload}
+        
+        case 'ALL' :
+             return initial
+        default:
+            break;
```

Comment:
Suggestions I don't think you need to write break here.

Reply:
thank you for point it out .I forgot to remove it